packages/server/src/db/migrations/20200831133732_discounts.ts

```diff
@@ -8,7 +8,7 @@ export async function up(knex: Knex): Promise<void> {
     table.date("startDate").notNullable();
     table.string("amount").notNullable();
     table.string("type").notNullable();
-    table.bigInteger("duration").notNullable();
+    table.date("endDate").notNullable();
```

Comment:
Avoid changing any of written migrations. If you want to change a column type, then make an additional migration. This change won't be deployed to production, because this migration is already applied to DB.